---
title: working title Compatibility system and stigma size are the main predictors of heterospecific pollen effect
author:
- affilnum: '1,2,3'
  name: Jose B. Lanuza, Ignasi Bartomeus, Tia-Lynn Ashman, Romina Rader
  email: barragansljose@gmail.com
affiliation:
- affil: US Environmental Protection Agency, Office of Research and Development, National
    Health and Environmental Effects Research Laboratory, Atlantic Ecology Division,
    27 Tarzwell Drive  Narragansett, RI, 02882, USA
  affilnum: 1
- affil: Big Name University, Department of R, City, BN, 01020, USA
  affilnum: 2
- affil: Estacion Biologica de Donana (EBD-CSIC), E-41092 Sevilla, Spain
  affilnum: 3
output:
  pdf_document:
    fig_caption: yes
    fig_height: 7
    fig_width: 7
    includes:
      in_header: header.tex
    keep_tex: yes
    number_sections: no
    template: manuscript.latex
  word_document: null
  html_document: null
capsize: normalsize
classoption: a4paper
csl: ecology.csl
documentclass: article
fontsize: 11pt
footerdate: yes
geometry: margin=1in
linenumbers: yes
bibliography:
- references.bib
- knitcitations.bib
spacing: doublespacing
abstract: no
---


```{r setup, include=FALSE, cache=FALSE, message = FALSE}

library("knitr")
library("ggplot2")
library("extrafont")

#opts_knit$set(root.dir=normalizePath('../'))

### Chunk options: see http://yihui.name/knitr/options/ ###

## Text results
opts_chunk$set(echo = TRUE, warning = TRUE, message = TRUE, include = TRUE)

## Code decoration
opts_chunk$set(tidy = TRUE, comment = NA, highlight = TRUE)

## Cache
opts_chunk$set(cache = 2, cache.path = "output/cache/")

## Plots
opts_chunk$set(fig.path = "output/figures/")


```


```{r knitcitations, echo=FALSE, cache = FALSE}
library(knitcitations)
cleanbib()   
cite_options(citation_format = "pandoc")
```


\singlespace

\vspace{2mm}\hrule

Pollinator sharing can have negative consequences for plant fitness with the arrival of foreign pollen. However, the costs of heterospecific pollen are not yet well understood. We conducted a glasshouse experiment to understand how phylogenetic relatedness and plant traits mediate the impacts of heterospecific pollen transfer. We conducted 1800 crosses by experimentally transferring pollen (50% and 100% ratio) with reciprocal crosses between 10 species belonging to three different families: Brassicaceae, Solanaceae and Convolvulaceae. Seed set was used as proxy of plant fitness. We found that for 65% of the treatments with 50% mix reduced seed set. Moreover, the reduction in seed set was dependent on the degree of relatedness and reproductive traits of the pollen recipient and not the pollen donor. Our results show that certain traits, particularly compatibility system, are critical in understanding the costs of heterospecific pollen.
\vspace{3mm}\hrule

*Keywords*: heterospecific pollen, plant reproduction, fitness, interspecific competition, phylogenetic distance. 

\doublespace

\bleft

# INTRODUCTION


In most ecosystems, plant species normally coexist and share their floral visitors with other species @waser1996. From the plants’ perspective, pollinator sharing can be positive for some plants @carvalheiro2014 or negative for others @pauw2013, depending on the facilitation gradient. An increasing number of visits often correlates with higher chances of fertilization @engel2003. However this is not always the case, among these possible flower visitors there are also nectar robbers and pollen thieves @inouye1980; @magrach2017. Receiving both sufficient quantity and quality deposited on the stigma is thus highly relevant to the pollination success of the plant @aizen2007. 

By visiting many plant species, many pollinators are responsible for conspecific pollen loss and the transport of foreign pollen, both of which can have important detrimental effects on species fitness @morales2008;  @ashman2013; @arceo2016. Foreign pollen arrival can play an important role in plant species fitness but outcomes are variable and appear to be context dependent as there is not always a decrease in fitness @morales2008. Some of this variation is likely due to the enormous variability of foreign pollen transferred across systems ranging from 0 to 75 percent. However, most studies report ranges of heterospecific pollen between 0 and 20 percent of the total pollen load @bartomeus2008 @montgomery2012; @ashman2013; @fang2013, yet even these relatively low amounts of heterospecific pollen transferred can decrease fitness greatly @thomson1982. While we now have some understanding of the impacts of heterospecific pollen quantity, we have less understanding of other factors that could be driving the variation in impacts upon fitness.  @ashman2013 postulated the first predictive framework that identifies a need to understand how plant traits might mediate heterospecific pollen effect, whereby mating system and pollen size were predicted to potentially mediate the impact of foreign pollen transfer on plant fitness. This concept is supported by specific case studies, such as @tong2016 that demonstrate an asymmetrical effect in 6 species of *Pedicularis* whereby the pollen of long styled species was able to grow the full length of the style on short styled species but not vice versa. While this suggests that the impacts of heterospecific pollen may differ among pollen donor and recipient, few studies have been conducted to ascertain whether this pattern is in fact a general trend or to identify the extent to which other plant traits are critical to heterospecific pollen impacts.

Plant traits are crucial to understand heterospecific pollen effect but the multifactorial nature of the traits that are involve in the pollen-pistil interaction make difficult to unravel what are the main traits in driving the effect. These traits can be seen from a male perspective of both donor and recipient where pollen size, pollen aperture number and pollen allelopathy are key components to understand the outcome of foreign pollen arrival @murphy1995; @ashman2013. In @ashman2013 small pollen is predicted to cause a greater fitness decrease, although this can be true there are also other possibilities to consider which can obscure a predictive framework like big pollen can clogg small stigmas with fewer pollen grains, bigger stigmas are less likely to be clogged by small pollen grains and bigger pollen can outcompete smaller pollen grains due to faster pollen tube growth rate @williams1990. Moreover, to understand the different mechanical or chemical effects of pollen also the female traits of the pollen recipient have to be considered, from the literature these main traits are: stigma size, style length, number of ovules, incompatibility system and also a structural trait such as flower morphology @montgomery2012; @ashman2013; @tong2016. For example, greater stigmatic area is positively correlated with greater heterospecific pollen deposition @montgomery2012 and therefore possibly with an increase in negative effect. For species that are self-incompatible the barriers towards heterospecific pollen are stronger than self-compatible species @ashman2013. Nonetheless, an effect of foreign pollen is a bit obscured by the variability within species, however species that are strong selfers or strong outcrossers have less variablity in mating systems and predictions of effect could be more realistic (see figure 1 from @whitehead2018). Although past research has progress in the understanding of what traits can mediate the effect as we have shown here, there are multiple traits involved and multiple possible scenarios still to be explored empirically for a full understanding of the importance of heterospecific pollen effect in nature.

For the understanding at what level or intensity the interference of pollen can occur is important to consider the relatedness of the interacting species. Closely related species are more likely to have similar traits @letten2015. The similarity in traits between closely related species can lead to higher chances of ovule usurpation/abortion @arceo2011 and therefore studies predict and show a greater negative effect of closely related species @ashman2013; @arceo2016. Few studies however, have focused on the impacts of heterospecific pollen on fitness of distantly related species @galen1989; @neiland1999. Despite the fact that far related species are also able to decrease species fitness (REFS). Yet, most insects and most stigmas have been found to carry multiple species of foreign pollen with little attention to degree of relatedness (Arceo-Gómez and Ashman (2016); Fang and Huang (2013). Understanding the role of foreign pollen from distantly related species thus deserves greater attention. The relatedness of foreign pollen gives a first snapshoot of where the pollen competition can occur and therefore could be a proxy of effect. With that said, @arceo2016 is the only work which has proven a greater effect of close related species through a meta-analysis but with low sample sizes and lack of significance. Therefore, there is a need to study the effect of heterospecific pollen of far and close related species at community level beyond single pairwise interactions.

Heterospecific pollen studies in nature have the complexity added of great environmental variability which can lead to confounding interpretations. Moreover, the great diversity of floral structures and small sizes of some of them such as Asteraceae species make tedious or almost impossible the studies of heterospecific pollen on the field. For this reason, we investigated how floral reproductive traits and relatedness mediate the impact of heterospecific pollen by creating an artificial co-flowering community in a glasshouse with 10 species belonging to three different families with different traits. Our study tries to answer the following questions: (i) Do heterospecific pollen reduce seed set? To what extent do (ii) floral reproductive traits and (iii) relatedness, mediate the impacts of heterospecific pollen on seed set.


# METHODS

The study was conducted in a glasshouse at University of New England (Armidale, Australia) from November 2017 to March 2018. Rooms were temperature controlled depending on the requirements of the species with day and night temperature differences. The experimental design had species from three different families: Solanaceae, Brassicaceae and Convolvulaceae (**Table 1**). The species of the study had different reproductive traits and different degree of relatedness where the reciprocal crosses between species allowed us to have multiple different scenarios of both traits and relatedness. Moreover, the species selected had fast life cycle and low structural flower complexity in order to perform the pollination treatments and grow the different species from seeds. For the purpose of the experiment all the species where considered as pollen recipient and as pollen donor (see interaction matrix, fig 2). Species were watered once or twice per day and fertilized weekly (NPK 23: 3.95: 14).

\newpage
**Table 1**
```{r, echo=FALSE, message=FALSE, warning=FALSE, cache=FALSE}
library(kableExtra)
Family <- c(rep("Brassicaceae",4), rep("Convolvulaceae", 2), rep("Solanaceae",4))

Genus <- c("Brassica", "Brassica", "Eruca", "Sinapis", "Ipomoea", "Ipomoea", "Capsicum", "Petunia", "Solanum", "Solanum")

Species <- c("Brassica rapa", "Brassica oleracea", "Eruca versicaria", "Sinapis alba", "Ipomoea aquatica", "Ipomoea purpurea", "Capsicum annuum", "Petunia integrifolia", "Solanum lycopersicum", "Solanum melongena")

table<- data.frame(Family, Genus, Species)
kable(table, format="markdown")
```

\newpage
```{r, echo=FALSE, message=FALSE, warning=FALSE, cache=FALSE, out.width='100%', fig.cap="Phylogenetic tree of the ten species used in the experiment from three different families from bottom to top: Solanaceae, Convolvulaceae and Brassicaceae."}
knitr::include_graphics('images/phylo_image.png')
```


**Hand-pollination**

Foreign pollen effect was studied through two different treatments, one with 50% conspecific pollen and 50% heterospecific pollen and a second one with 100% foreign pollen (N=10) in order to see if foreign pollen can trigger fruit production by itself or even seeds through ovule usurpation. Therefore, 180 different combinations were performed with N=10 per combination. Seed set was the proxy of effect for all our treatments. Moreover, hand cross pollination (between individuals of the same species), hand self pollination, apomixis (bagged emasculated flowers) and natural selfing were tested for each species (N=10). For the treatments with foreign pollen and hand cross pollination, flowers were emasculated the day prior anthesis and hand pollinated next day with a toothpick. Hand-pollination was conducted with 3-4 gentle touches on the stigma surface. For each species 20 anthers were collected and their pollen counted with a hemocytometer, each anther was counted 4 times and then an average of these counts was performed. Once, the average number of pollen grains per anther was known, the proportion of anthers per mix was calculated in order to achieve a 50-50% mix.  In order to confirm that the treatments applied were the desire proportions, the total stigmatic load of pollen was counted and the proportions calculated between the two species of the mix. Therefore, 20 different heterospecific crosses 50-50% were counted for all the focal species with crosses of just one species of the other two families (N=3).

**Traits and evolutive distance**

The traits measured for each species were pollen per anther, number of ovules and stigma, style, ovary width and length. For the stigma, the stigmatic area was also measured and moreover the stigmas were divided in wet/dry type. All the morphometrical measurements were performed with a stereophotomicroscope. Pollen was counted for 20 anthers of each species with 4 replicates per sample with an hemocytometer. Previously, anthers were squashed on a known solution with the pippete tip and homogeneize with a vortex for 30 seconds. Ovule number was counted with the help of a stereomicroscope and a small grid over a petri dish from 15 randomly selected flowers. Fruits per number of flowers treated were counted for Solanaceae species and for the rest, the number of seeds produced per average number of ovules of each species. Levels of self incompatibility were estimated by dividing the fruit set of hand self pollination by hand cross pollination @lloyd1992. 

**Analysis**

To evaluate heterospecific pollen effect on seed production we performed linear mixed models. The distinct heterospecific pollination treatments were compared through relevelling each variable with the cross pollination treatment which was our control for optimum seed production for all the species. The different replicates of each treatment were considered as random effects. Seed production was scaled for all the species with mean 0 and standard deviation of 1 prior to the analysis. All the analysis were conducted with the statistical language `R` `r citep(citation())`.

In order to test the relative effect of traits on seed production with foreign pollen we performed Mantel test between the assymetrical matrix of heterospecific pollen effect (10 by 10 matrix) with the different distance matrices of traits (Euclidean distances). Heterospecific pollen effect was obtained through the subtraction of seed production by hand cross pollination minus seed production of the different heterospecific pollen treatments. To find a model with the best explanatory traits we used the function **bioenv** from R. We also conducted Mantel test between the matrix of heterospecific pollen effect and the distance matrix from all the traits. Moreover, we explored also the correlation between traits and heterospecific pollen effect through generalized mixed models where the response variable was heterospecific pollen effect and the explanatory variable the different traits. In addition, we tested the correlation between the total amount of pollen deposited on the stigma with heterospecific pollinations and the stigma size through Pearson's correlation. 

We conducted mantel test to check for correlations between heterospecific pollen effect and phylogenetic distance. Due to improvents in statistical power we used the square root of the phylogenetic distance (Letten & Cornwell 2014). Two different phylogenetic distances were used from two kinds of markers: 1) Internal transcribed spacer (ITS) and 2) ribulose-bisphosphate carboxylase (RBCL). The sequences were obtained from GenBank (https://www.ncbi.nlm.nih.gov/genbank/, accessed 20 Oct. 2018). The sequences were aligned with clustal omega and the pairwise evolutive distances calculated with MEGA7.

[Phylogenetic signal of traits?](Jose)

[Add pollen on stigma counts to methods](Jose)

[Next bit to results, also divided it with ]

Total pollen deposited on stigmas was significantly correlated with the stigmatic area, pearsons correlation=0.57 and p-value=0.008. Proportions of pollen talk about it to 50-50 mix? Talk aboout this in methods too.

Add also plot with fruit production and pot it in appendix.Also the plots of ratios to appendices.

Add effect sizes to analysis!!

# RESULTS

Results of hand cross-pollination, self hand-pollination, natural selfing and apomixis are presented in **Figure 1** (see appendix 1 for table with values). Heterospecific pollen reduced seet set significatively with the 50-50% heterospecific pollen treatments for 65% of the pairwise interactions p<0.05. Moreover, effect sizes differ across species, see **Figure 2** where species like *S. lycopersicum* had a large effect size and other's like *E. sativa* a small or null effect size.
 

Across families we found a very similar effect but when species where look at species level they respond differently even within the same family [rephrase and maybe test statistically?](), for instance for two species of the Brassicaceae family *Brassica oleracea* and *Eruca versicaria* we found very contrasting effects of foreign pollen where for the first one, all donors reduce seed set significatively and for the second, just two species did out of nine. The 100% foreign pollen treatments barely produced seeds or fruits [and just for *Sinapis alba* we did not find significant differences between the hand cross pollination and one treatment with pollen from a confamilial- IB Unclear](). Solanaceae species with berry fruit type developed small fruits or even normal fruits in some cases [under which treatment](). *S. lycopersicum* seems to produced small fruits (35% of the treatments) independently of pollen and pollen donor due to also apomictic treatments did, never normal size. *C. annuum* produced some fruits (9%) of both small and normal size and finally *S. melongena* produced seedless normal fruits with just confamilial pollen (3%), for both species seems that fruit formation was induced by pollen on the stigma because of lack of fruit production with treatments that tested for apomixis.[clarify this descriptive statistics part- Also a figure with a summary of the treatments effect would be cool, or at least in the appendices]()

\newpage
```{r, echo=FALSE,fig.align='center',message=FALSE, warning=FALSE, cache=FALSE,fig.height=4,fig.width=7, fig.cap="Barplot with the different treatments that provide information of the reproductive biology of the ten species. The y axis is the proportion of ovules converted to seed in percentage. The different treatments (N=10) which are presented in the legend are, hand cross pollination, hand self pollination, natural selfing and apomixis. More information about these treatments can be found in Methods and Appendices."}

load("r_biology.RData")

ggplot(bio,aes(x=Species,y=value,fill=variable))+
  geom_bar(stat="identity",position="dodge")+scale_fill_grey(start = 0, end = 0.85,name="Treatments") + 
  theme_minimal()+xlab("Species")+ylab("Seed-setting rate (%)")+labs(title="Reproductive biology test")+
  theme(axis.text.x = element_text(angle = 45, hjust = 1, face= "italic",family = ""), legend.text=element_text(size=7), legend.title=element_text(size=8), axis.text.y  = element_text(size = 10))+
  scale_y_continuous(minor_breaks = seq(0 , 100, 5), breaks = seq(0, 100, 10))

```

\newpage
```{r, echo=FALSE, message=FALSE, warning=FALSE,cache=FALSE,fig.height=2.5,fig.width=6,fig.cap="Average effect size and confidence intervals (95%) of the 9 heterospecific treatments of 50% pollen to the different 10 focal species coloured by family."}

library(effsize)
library(dplyr)
library(ggplot2)
library(ggpubr)
library(cowplot)
library(gridExtra)
library(egg)
library(magrittr)
library(multipanelfigure)
library(grid)

load("effect_size_species/effect_size_total.RData")

 ggplot(all, aes(Family,Cohen_d,colour=Family_1), size=10) + theme_bw(base_size=10)+theme(axis.text.y = element_text(face = "italic"))+ geom_point( aes(colour=Family_1),size=2) +
  geom_errorbar(aes(x = Family, ymin = Lower, 
ymax = Upper,colour=Family_1), width = 0.2, size=0.8, alpha=0.5)+
  scale_color_manual(name  ="Family",values=c( "#0072B2", "#009E73", "#D55E00"), labels=c("Solanaceae", "Convolvulaceae","Brassicaceae"))+
  scale_fill_manual(name  ="Family",values=c( "#0072B2", "#009E73", "#D55E00"), labels=c("Solanaceae", "Convolvulaceae","Brassicaceae"))+
  xlab("") + ylab("Hedges' g") + rotate()+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+scale_alpha(guide = 'none')
```



\newpage
```{r, echo=FALSE, message=FALSE, warning=FALSE,cache=FALSE,fig.height=8,fig.width=7,fig.cap="Effect sizes for the 10 different species. The different families appear with different colours, when a species was focal was coloured differently from its family."}

library(effsize)
library(dplyr)
library(ggplot2)
library(ggpubr)
library(cowplot)
library(gridExtra)
library(egg)
library(magrittr)
library(multipanelfigure)
library(grid)

#SOME
load("effect_size_species/all_effect_size.RData")

#BROL
load("effect_size_species/brol_effect_size.RData")
p1<- ggplot(brol_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_text(face = c('italic', 'italic', 'italic', 'italic', 'italic','italic', 'italic', 'italic', 'italic', 'bold')),plot.margin = margin(t=0, l=0))+geom_point(alpha=c(1,1,1,1,1,1,1,1,1,0.5),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=c(0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5),size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+annotate("text", x=10, y=-6, label= "A)")

#BRRA
load("effect_size_species/brra_effect_size.RData")
pp2<- ggplot(brra_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_blank())+ geom_point(alpha=c(1,1,1,1,1,1,1,1,0.5,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+annotate("text", x=10, y=-6, label= "B)")

#ERSA
load("effect_size_species/ersa_effect_size.RData")
p3<- ggplot(ersa_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(alpha=c(1,1,1,1,1,1,1,0.5,1,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+annotate("text", x=10, y=-6, label= "C)")+theme(axis.text.y = element_text(face = c('italic', 'italic', 'italic', 'italic','italic','italic', 'italic', 'bold', 'italic', 'italic')))


#SIAL
load("effect_size_species/sial_effect_size.RData")
p4<- ggplot(sial_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_blank())+ geom_point(alpha=c(1,1,1,1,1,1,0.5,1,1,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+ annotate("text", x=10, y=-6, label= "D)")

#IPAQ
load("effect_size_species/ipaq_effect_size.RData")
p5<- ggplot(ipaq_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_text(face = c('italic', 'italic', 'italic', 'italic','italic','bold', 'italic', 'italic', 'italic', 'italic')),plot.margin = margin(t=0, l=0))+geom_point(alpha=c(1,1,1,1,1,0.5,1,1,1,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+annotate("text", x=10, y=-6, label= "E)")


#IPPU
load("effect_size_species/ippu_effect_size.RData")
p6<- ggplot(ippu_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+
  theme(axis.text.y = element_blank())+geom_point(alpha=c(1,1,1,1,0.5,1,1,1,1,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+ annotate("text", x=10, y=-6, label= "F)")

#CAAN
load("effect_size_species/caan_effect_size.RData")
p7<- ggplot(ersa_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_text(face = c('italic', 'italic', 'italic', 'bold', 'italic','italic', 'italic', 'italic', 'italic', 'italic')))+ geom_point(alpha=c(1,1,1,0.5,1,1,1,1,1,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+annotate("text", x=10, y=-6, label= "G)")

#PEIN
load("effect_size_species/pein_effect_size.RData")
p8<- ggplot(caan_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_blank(),plot.margin = margin(t=0, l=0))+ geom_point(alpha=c(1,1,0.5,1,1,1,1,1,1,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+annotate("text", x=10, y=-6, label= "H)")

#SOLY
load("effect_size_species/soly_effect_size.RData")
p9<- ggplot(soly_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_text(face = c('italic', 'bold', 'italic', 'italic', 'italic','italic', 'italic', 'italic', 'italic', 'italic')),plot.margin = margin(t=0, l=0))+ geom_point(alpha=c(1,0.5,1,1,1,1,1,1,1,1),size=2,show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(alpha=0.5,size=0.6,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+ annotate("text", x=10, y=-6, label= "I)")

#SOME
load("effect_size_species/some_effect_size.RData")

p10<- ggplot(some_effect_size, aes(Species_1,Cohen_d, size=10)) + theme_bw(base_size=10)+theme(axis.text.y = element_blank())+ geom_point(size=2,alpha=c(0.5,1,1,1,1,1,1,1,1,1),show.legend = FALSE,aes(color=factor(Family))) +geom_errorbar(size=0.6,alpha=0.5,show.legend=FALSE, aes(x = Species_1, ymin = Lower, ymax = Upper,color=factor(Family)))+scale_color_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  scale_fill_manual("Family",values=c("#D55E00", "#009E73", "#0072B2"))+
  xlab("") + ylab("Hedges' g") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")+ annotate("text", x=10, y=-6, label= "J)")

 #grid.arrange(p1, p5, p3, p4, nrow=2,ncol=2,top="Solanaceae")
grid.newpage()
 grid.draw(arrangeGrob(cbind(ggplotGrob(p1),ggplotGrob(pp2), size="last"),cbind(ggplotGrob(p3),ggplotGrob(p4),size="last"),cbind(ggplotGrob(p5),ggplotGrob(p6), size="last"),cbind(ggplotGrob(p7),ggplotGrob(p8), size="last"),cbind(ggplotGrob(p9),ggplotGrob(p10), size="last"),nrow=5))

```

\newpage

```{r, echo=FALSE, message=FALSE,warning=FALSE,cache=FALSE,fig.height=8,fig.width=7,fig.cap="A"}
library(effsize)
library(dplyr)
library(ggplot2)
library(ggpubr)
library(cowplot)
library(gridExtra)
library(egg)
library(magrittr)
library(multipanelfigure)
library(grid)


load("effect_size_species/some_effect_size_family.RData")

pp1<- ggplot(some_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")



load("effect_size_species/soly_effect_size_family.RData")

pp2<- ggplot(soly_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")


load("effect_size_species/sial_effect_size_family.RData")

p3<- ggplot(sial_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")


load("effect_size_species/pein_effect_size_family.RData")

p4<- ggplot(pein_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")


load("effect_size_species/ippu_effect_size_family.RData")

p5<- ggplot(ippu_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")

load("effect_size_species/ipaq_effect_size_family.RData")

p6<- ggplot(ipaq_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")

load("effect_size_species/ersa_effect_size_family.RData")

p7<- ggplot(ersa_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")

load("effect_size_species/caan_effect_size_family.RData")

p8<- ggplot(caan_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")

load("effect_size_species/brra_effect_size_family.RData")

p9<- ggplot(brra_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")


load("effect_size_species/brol_effect_size_family.RData")

p10<- ggplot(brol_effect_size, aes(Family,Cohen_d, size=10)) + theme_bw(base_size=10)+ geom_point(size=2,show.legend = FALSE,aes(color=factor(Family))) +
  geom_linerange(size=0.6,linetype=1,show.legend=FALSE, aes(x = Family, ymin = Lower, 
ymax = Upper,color=factor(Family)))+
  scale_color_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  scale_fill_manual("Family",values=c("#E69F00", "#0072B2", "#009E73", "#D55E00"))+
  xlab("") + ylab("Cohen's d") + rotate(ylim = c(-6, 2))+guides(fill=FALSE)+
  geom_hline(yintercept=0, linetype="dashed", color = "black")

 #grid.arrange(p1, p5, p3, p4, nrow=2,ncol=2,top="Solanaceae")
grid.newpage()
 grid.draw(arrangeGrob(cbind(ggplotGrob(pp1),ggplotGrob(pp2), size="last"),cbind(ggplotGrob(p3),ggplotGrob(p4),size="last"),cbind(ggplotGrob(p5),ggplotGrob(p6), size="last"),cbind(ggplotGrob(p7),ggplotGrob(p8), size="last"),cbind(ggplotGrob(p9),ggplotGrob(p10), size="last"),nrow=5))
```


Maybe for this plot I have to do hedges'g instead of cohean because of different sample sizes when I group the treatments per family. Check

\newpage

Mantel test indicates that a possible?? [It exists!]() correlation exist between heterospecific pollen effect and the evolutive relative distances, for ITS and RBCL markers we had r coefficients of 0.29 and 0.25 respectively p<0.05 [think on a figure - maybe using NMDS](). Moreover, Mantel test indicates that also a possible?? correlation between stigma width and stigma type exist (stats??). Trait correlations were also explored with GLMM

[I have done it at the moment just for Compatibility system](Jose)
[Also I have to fix from mixed linear model to GLMM, just realize that](Jose)


\newpage
```{r, echo=FALSE, message=FALSE, warning=FALSE, cache=FALSE,fig.height=4,fig.width=5, fig.cap="The effect of heterospecific pollen (scaled see set) is represented in function of the compatibility system (self/cross*100) for the the different species. Each coulored dot represents the interaction of a focal species with a different pollen donor."}
load("myEnv.RData")
library(nlme)
library(lme4)
library(ggplot2)
library(cowplot)
model2=lme(hp_effect~compatibility, data=ALL, random=~1|indv) #I don't get the random here. I think you need a better analysis.
ggplot(ALL, aes(x=compatibility, y=hp_effect)) + 
  geom_jitter(width=1.5,aes(colour = Focal),size=4)+
  geom_abline(aes(intercept=`(Intercept)`, slope=compatibility), as.data.frame(t(fixef(model2))))+theme_cowplot()

```


[Compatibility index don´t multiply per 100 from Lloyd](Jose)

\newpage


# DISCUSSION
 
Discussion




Herbs vs tress, annual vs perennial... Many flowers vs few flowered species; structural composition on a system

 What are the implications of the findings?










Ideas about pollen size in heterospecific pollen effect. (still have to develop it more...)

Let's classify pollen size in three groups in order to understand the interaction between pollen donor and recipient:
1) Donor pollen size < Recipient pollen size
2) Donor pollen size = Recipient pollen size
3) Donor pollen size > Recipient pollen size

Now I try to develop each part

1) Donor pollen size < Recipient pollen size

Effect:

- Donor's pollen could clogg the stigma

- Chemical inhibition

Traits associated with bigger pollen of the recipient:

- Recipient's pollen have faster pollen tube growth (example with my data)

- Reduction in number of ovules (Also with my species)

- Big differences in pollen size can be traduced in low relatednes therefore less likely of pollen germination on a far related stigma.


2) Donor pollen size = Recipient pollen size

- Very relatedness dependant this point

- Similar probabilities of taken space on the stigma


3) Donor pollen size > Recipient pollen size

Effect: 

-In small stigmas big pollen grains can occupy great part of the stigmatic area.

-small pollen grains can get embeded

IB: Think also on using tree analysis to test if hp effect depends on complex trait combinations. Tree analysis are great when two different strategies lead to the same outcome. This would never been pick up by GLMs. The r package is party{} . You can see an example applied to birds is Sol et al 2010 Science. Ask me if you want more details or code examples.


# CONCLUSIONS



# ACKNOWLEDGEMENTS



# REFERENCES

```{r write_citations, cache=FALSE, include=FALSE}
write.bibtex(file="knitcitations.bib")
```

<div id = "refs"></div>

# APPENDIX

1.

**Table S1**. Perecentage of seeds produced per ovule for the ten species used in the experiment. The treatments presented are hand cross pollination, hand self pollination, natural selfing and apomixis (emasculated flowers).
```{r, echo=FALSE, message=FALSE, warning=FALSE, cache=FALSE}
library(kableExtra)
load("table_treatments.RData")
Species <- c("Brassica oleracea", "Brassica rapa", "Eruca versicaria", "Sinapis alba", "Ipomoea aquatica", "Ipomoea purpurea", "Capsicum annuum", "Petunia integrifolia", "Solanum lycopersicum", "Solanum melongena")
Cross <- cross
Self <- self
Natural_selfing <- nat_self
Apomixis <- apo
table<- data.frame(Species, Cross, Self, Natural_selfing,Apomixis)
kable(table, format="markdown")
```

2.

**Figure S1**. Pollen ratios separated by family A) Brassicaceae, B) Solanaceae, C) Convolvulaceae. Pollen was counted for crosses with species of the other two families (N=3). To prepare these ratios all the pollen grains on the stigma were counted.

```{r, echo=FALSE, message=FALSE, warning=FALSE, cache=FALSE}
load("pollen_ratio.RData")

ggplot(pollen_ratios_brassicaceae, aes(x=spp, y=ratio, fill=variable)) +
  geom_bar(stat='identity', position='dodge') +labs(title="A) Brassicaceae") +
  theme(axis.text.x=element_text(angle=45,hjust=1, face="italic"), legend.title = element_blank()) + scale_fill_grey(labels=c("Recipient","Donor")) + 
  geom_hline(yintercept = 50) + xlab("") + ylab("Pollen Ratio")

ggplot(pollen_ratios_solanaceae, aes(x=spp, y=ratio, fill=variable)) +
  geom_bar(stat='identity', position='dodge') +labs(title="B) Solanaceae") +
  theme(axis.text.x=element_text(angle=45,hjust=1, face="italic"), legend.title = element_blank()) + scale_fill_grey(labels=c("Recipient","Donor")) + 
  geom_hline(yintercept = 50) + xlab("") + ylab("Pollen Ratio")

ggplot(pollen_ratios_convolvulaceae, aes(x=spp, y=ratio, fill=variable)) +
  geom_bar(stat='identity', position='dodge') +labs(title="C) Convolvulaceae") +
  theme(axis.text.x=element_text(angle=45,hjust=1, face="italic"), legend.title = element_blank()) + scale_fill_grey(labels=c("Recipient","Donor")) + 
  geom_hline(yintercept = 50) + xlab("") + ylab("Pollen Ratio")

```

\eleft

\clearpage


\listoftables


\newpage


\newpage

\clearpage

\listoffigures


\newpage

\newpage

\blandscape

\elandscape

\clearpage

